paystell-frontend/src/app/settings/page.tsx

```diff
@@ -0,0 +1,137 @@
+      reader.readAsDataURL(file); //base 64
+    }
+  };
+
+  return (
+    <div className="flex flex-col items-center justify-center min-h-screen bg-gray-100 py-6 p-8">
+      <div className="bg-white rounded-lg shadow-md w-full p-8">
+        <h1 className="text-2xl font-bold mb-6 text-center">Settings</h1>
+
+        <form onSubmit={handleSubmit} className="space-y-6">
+          {/* Avatar Section */}
+          <div className="flex flex-col items-center">
+            <div className="relative">
+              <Image
+                src={logo || "/default-avatar.png"}
+                alt="Profile Picture"
+                width={120}
+                height={120}
+                className="rounded-full border"
+              />
+              <button
+                type="button"
+                onClick={() =>
+                  document.getElementById("logoInput")?.click()
+                }
+                className="absolute bottom-0 right-0 bg-blue-500 text-white p-2 rounded-full"
+              >
+                <Edit/>
+              </button>
+            </div>
+            <input
+              id="logoInput"
+              type="file"
+              className="hidden"
+              onChange={handleLogoChange}
+            />
+          </div>
+
+          {/* Name Input */}
+          <div>
+            <Label htmlFor="name">Name</Label>
+            <Input
+              id="name"
+              type="text"
+              value={name}
+              onChange={(e) => setName(e.target.value)}
+              placeholder="Enter your name"
+              className="mt-2"
+            />
+            {errors.name && (
+              <p className="text-red-500 text-sm">{errors.name}</p>
+            )}
+          </div>
+
+          {/* Description */}
+          <div>
+            <Label htmlFor="description">Description</Label>
+            <Textarea
+              id="description"
+              value={description}
+              onChange={(e) => setDescription(e.target.value)}
+              placeholder="Write something about yourself"
+              className="mt-2"
+            />
+          </div>
+
+          {/* Buttons */}
+          <div className="flex justify-between">
+            {isEditing ? (
+              <>
+                <Button
+                  type="button"
+                  className="bg-gray-500"
+                  onClick={() => setIsEditing(false)}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  type="submit"
+                  className="bg-blue-500"
+                >
+                  Save
+                </Button>
+              </>
+            ) : (
+              <Button
+                type="button"
+                className="bg-blue-500 w-full"
+                onClick={() => setIsEditing(true)}
+              >
+                Edit Profile
+              </Button>
```

Comment:
remove className, use variant 
```suggestion
              <Button
                type="button"
                variant="default"
                onClick={() => setIsEditing(true)}
              >
                Edit Profile
              </Button>
```